app/controllers/api/v1/posts_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::PostsController < ApplicationController
+  load_and_authorize_resource
+
+  def index
+    @posts = Post.all.order('created_at')
+    render json: { success: true, data: { posts: @posts } }, status: :ok
+  end
+end
```

Comment:
Good job working on the project requirements so far. But whenever  I try to delete a post its throwing an error. Please kindly look at the error below:
![Screenshot 2023-10-13 at 8 03 52 PM](https://github.com/tchasinga/Blog-app-ruby/assets/79506105/4a046452-3fed-49e3-93e6-49f527deb150)
 It would be nice if you fixed this error.

Secondly I notice the added content is being duplicated kindly look at the screen shot below:

![Screenshot 2023-10-13 at 8 03 17 PM](https://github.com/tchasinga/Blog-app-ruby/assets/79506105/b28c3816-e57e-4183-91ac-76e35c0e49e6)
![Screenshot 2023-10-13 at 8 03 45 PM](https://github.com/tchasinga/Blog-app-ruby/assets/79506105/753bd4ef-cd19-471a-8b22-cff0365ba90d)

It would be nice if you fixed this issue.